src/test/java/api/loans/DeclareClaimedReturnedItemAsMissingApiTests.java

```diff
@@ -0,0 +1,125 @@
+import static api.support.matchers.ValidationErrorMatchers.hasMessage;
+import static api.support.matchers.ValidationErrorMatchers.hasParameter;
+import static org.folio.circulation.domain.representations.ChangeItemStatusRequest.COMMENT;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION_COMMENT;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.ClaimItemReturnedRequestBuilder;
+import api.support.builders.DeclareClaimedReturnedItemAsMissingRequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class DeclareClaimedReturnedItemAsMissingApiTests extends APITests {
+  private InventoryItemResource item;
+  private String loanId;
+
+  @Before
+  public void setUpItemAndLoan() {
+    item = itemsFixture.basedUponSmallAngryPlanet();
+    loanId = loansFixture.checkOutByBarcode(item, usersFixture.charlotte())
+      .getId().toString();
+  }
+
+  @Test
+  public void canDeclareItemMissingWhenClaimedReturned() {
+    final String comment = "testing";
+
+    claimItemReturnedFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
+      .forLoan(loanId)
+      .withItemClaimedReturnedDate(DateTime.now()));
+
+    claimItemReturnedFixture.declareClaimedReturnedItemAsMissing(
+      new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment(comment));
+
+    assertLoanIsClosed(comment);
+    assertItemIsMissing();
+  }
+
+  @Test
+  public void cannotDeclareItemMissingWhenIsNotClaimedReturned() {
+    final Response response = claimItemReturnedFixture
+      .attemptDeclareClaimedReturnedItemAsMissing(new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment("testing"));
+
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Item is not Claimed returned"),
+      hasParameter("itemId", item.getId().toString()))));
+  }
+
+  @Test
+  public void cannotDeclareItemMissingWhenLoanIsClosed() {
+    loansFixture.checkInByBarcode(item);
+
+    final Response response = claimItemReturnedFixture
+      .attemptDeclareClaimedReturnedItemAsMissing(
+        new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+          .forLoan(loanId)
+          .withComment("testing"));
+
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson(), hasErrorWith(hasMessage("Loan is closed")));
+    assertThat(response.getJson(), hasErrorWith(hasParameter("id", loanId)));
```

Comment:
Similarly to the above change, please can all of the validation error assertions be changed to be more specific